app/src/main/java/com/example/weatherandroidapp/contracts/WeatherGridContract.kt

```diff
@@ -2,6 +2,6 @@ package com.example.weatherandroidapp.contracts
 
 interface WeatherGridContract {
     interface ViewModel {
-        fun getWeatherInfo()
+        //fun getWeatherInfo()
```

Comment:
please do not push commented code. Uncomment this or delete it

Reply:
pucha jaja, done